share/spice/iplookup/iplookup.js

```diff
@@ -30,7 +37,7 @@
             templates: {
                 group: 'text',
                 options: {
-                    content: api_result.ddg.blacklists.length ? Spice.iplookup.content : null,
```

Comment:
@moollaza Ok so here's the reason I'm not using `getProperty` http://dev.mrchris.me/testing-getproperty.html 

It returns True for an empty array.